docs/docs/faq.md

```diff
@@ -2,16 +2,25 @@
 Common questions which can come up during development.
 
 ## DLL not found Exception
-If the binaries are installed on the system but are not found during runtime a `DllNotFoundException` is raised. In this case the names of the installed libraries probably don't match the expected names.
 
-The gir.core nuget packages are build against 3 different package sources:
+The `System.DllNotFoundException` can be thrown in the following cases:
+
+- required DLLs or Shared Objects (the Unix equivalent of DLLs) are not installed in the OS.
+
+- or you forgot to do a `Module.Initialize()` call before instantiating an object in a namespace like `Gtk`, `Gdk` or `GtkSource`.
```

Comment:
- Please don't talk to the reader directly / avoid words like `you`
- Remove empty lines between bullet points